perfkitbenchmarker/linux_benchmarks/cassandra_ycsb_benchmark.py

```diff
@@ -50,10 +50,9 @@
 
 # TODO: Add flags.
 REPLICATION_FACTOR = 3
+COLUMN_FAMILY = 'data'
```

Comment:
But from my understanding "column family" means "table" in Cassandra. Which means the code in 165 actually created a table named "data", but the new create-ycsb-table.cql created a table named "usertable"